ros/demos/chatbot/include/chatbot/tools/chatbot_tools.hpp

```diff
@@ -0,0 +1,68 @@
+#ifndef CHATBOT__TOOLS__CHATBOT_TOOLS_HPP
+#define CHATBOT__TOOLS__CHATBOT_TOOLS_HPP
+
+#include <rclcpp/rclcpp.hpp>
+#include <behavior_srvs/srv/chat_tools_function_call.hpp>
+#include <daemon_ros_client/msg/base_status.hpp>
+#include <std_msgs/msg/u_int8.hpp>
+#include <cloud_data/srv/current_local_weather_open_meteo.hpp>
+#include <cloud_data/srv/local_weather_forecast_open_meteo.hpp>
+#include <perceptions_analyzer/srv/perceive_objects.hpp>
+
+#include <memory>
+#include <nlohmann/json.hpp>
+#include <fmt/format.h>
+#include <algorithm>
+#include <ctime>
+
+class ChatbotTools
+{
+public:
+    ChatbotTools(std::shared_ptr<rclcpp::Node> node, rclcpp::CallbackGroup::SharedPtr callbackGroup);
+    ~ChatbotTools() = default;
+
+private:
+    rclcpp::Node::SharedPtr node_;
+    rclcpp::CallbackGroup::SharedPtr callbackGroup_;
+
+    rclcpp::Service<behavior_srvs::srv::ChatToolsFunctionCall>::SharedPtr service_volume_up_;
+    rclcpp::Service<behavior_srvs::srv::ChatToolsFunctionCall>::SharedPtr service_volume_down_;
+    rclcpp::Service<behavior_srvs::srv::ChatToolsFunctionCall>::SharedPtr service_get_weather_;
+    rclcpp::Service<behavior_srvs::srv::ChatToolsFunctionCall>::SharedPtr service_get_forecast_;
+    rclcpp::Service<behavior_srvs::srv::ChatToolsFunctionCall>::SharedPtr service_get_perceive_objects_;
```

Comment:
```suggestion
    rclcpp::Service<behavior_srvs::srv::ChatToolsFunctionCall>::SharedPtr service_get_perceived_objects_;
```
It starts with two active verbs in the other case, seems more grammatical this way. Or, `service_perceive_objects` would also work.